lib/node_modules/@stdlib/blas/base/dznrm2/test/test.ndarray.js

```diff
@@ -105,6 +105,82 @@ tape( 'the function computes the L2-norm', function test( t ) {
 
 	actual = dznrm2( 3, zx, 1, 0 );
 	isApprox( t, actual, expected, 2.0 );
+
+	zx = new Complex128Array([
+		1E150,  // 1
+		1E150,  // 1
+		1E150,  // 2
+		1E150,  // 2
+		1E150,  // 3
+		1E150,  // 3
+		1E150,  // 4
+		1E150   // 4
+	]);
+	expected = 2.828E+150;
```

Comment:
As in https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/lapack/base/dlassq/test/test.dlassq.js, you'll want to indicate how you arrived at the expected value. Additionally, we use `e` not `E`, per project conventions.